hathor/feature_activation/model/criteria.py

```diff
@@ -0,0 +1,81 @@
+#  Copyright 2023 Hathor Labs
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from typing import Any, Dict, Optional
+
+from pydantic import Field, validator
+
+from hathor import version
+from hathor.feature_activation import constants
+from hathor.feature_activation.feature import Feature
+from hathor.utils.pydantic import BaseModel
+
+
+class Criteria(BaseModel):
```

Comment:
I like this class. But I also think that each criteria should inherit from this class. This way, they can have their own attributes and methods according to their needs.

Reply:
As discussed in meeting, the `Criteria` class will only contain Feature Activation parameters and will not be subclassed

Reply:
Right. Thanks!